delfi_attributes.json

```diff
@@ -0,0 +1,1221 @@
+	},
+	{
+	   "DELFI_ID":2120,
+	   "LABEL":"Rampe und Wege mit Neigungen vorhanden/Zugang über Rampe",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":[
+				"footway",
+				"path"
+			 ],
+			 "incline":true
+		  }
+	   },
+	   "TAGS":{
+		  
+	   }
+	},
+	{
+	   "DELFI_ID":2121,
+	   "LABEL":"Verbindungsfunktion",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":[
+				"footway",
+				"path"
+			 ],
+			 "incline":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  "level":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2123,
+	   "LABEL":"Rampenbreite",
+	   "CONDITION":{
+		  "osm_element":"OpenWay",
+		  "osm_tags":{
+			 "highway":[
+				"footway",
+				"path"
+			 ],
+			 "incline":true
+		  }
+	   },
+	   "TAGS":{
+		  "width":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2124,
+	   "LABEL":"Rampenneigung",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":[
+				"footway",
+				"path"
+			 ],
+			 "incline":true
+		  }
+	   },
+	   "TAGS":{
+		  "incline":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2130,
+	   "LABEL":"Rolltreppe vorhanden",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":"steps",
+			 "conveying":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  
+	   }
+	},
+	{
+	   "DELFI_ID":2131,
+	   "LABEL":"Verbindungsfunktion der Rolltreppe",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":"steps",
+			 "conveying":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  "level":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2132,
+	   "LABEL":"Richtung der Rolltreppe",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":"steps",
+			 "conveying":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  "incline":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2133,
+	   "LABEL":"Wechselnde Richtung der Rolltreppe",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":"steps",
+			 "conveying":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  "conveying":[
+			 "forward",
+			 "backward",
+			 "reversible"
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2134,
+	   "LABEL":"Länge der Rolltreppe (Laufzeit)",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "highway":"steps",
+			 "conveying":true
+		  },
+		  "osm_element":"OpenWay"
+	   },
+	   "TAGS":{
+		  "duration":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2080,
+	   "LABEL":"Durchgangsbreite der Umlaufsperre/des Sperrelements/der Engstelle",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "barrier":"cycle_barrier",
+			 "cycle_barrier":[
+				"single",
+				"diagonal"
+			 ]
+		  }
+	   },
+	   "TAGS":{
+		  "maxwidth:physical":[
+			 
+		  ]
+	   }
+	},
+	{
+	   "DELFI_ID":2081,
+	   "LABEL":"Bewegungsfläche in, durch und aus der Engstelle",
+	   "CONDITION":{
+		  "osm_tags":{
+			 "barrier":"cycle_barrier",
+			 "cycle_barrier":[
+				"double",
+				"triple"
+			 ]
+		  }
+	   },
+	   "TAGS":{
+		  "opening":[
+			 
+		  ],
+		  "spacing":[
+			 
+		  ],
+		  "overlap":[
+			 
+		  ]
+	   }
+	}
```

Comment:
Please add them at the right position (DELFI attributes should be ordered)

Also I think we can omit `overlap`.

It's still not a perfect match, but it was always ambiguous what DELFI really wants here.